lib/smart_answer/calculators/coronavirus_employee_risk_assessment_calculator.rb

```diff
@@ -19,5 +22,11 @@ class CoronavirusEmployeeRiskAssessmentCalculator
     def workplace_should_be_closed_to_public
       WORKPLACES_CLOSED_TO_PUBLIC.include?(where_do_you_work) && !workplace_is_exception
     end
+
+    def workplace_opening_date
+      if WORKPLACES_OPENING_SOON_TO_PUBLIC.key?(where_do_you_work) && !workplace_is_exception
```

Comment:
Minor: this is redundant because a hash default all keys to `nil`.

Reply:
Ah righto! Good catch - I've simplified this method